### PR TITLE
Teams concept

### PR DESCRIPTION
**Pull request by danil-topchiy**
#9

### CODE REVIEW

**Comment by danil-topchiy** [comment]
I think we should consider making some generic meta-class for mixins like `LdapUserMixin, LdapTeamMixin, LdapFranchiseMixin, LdapDivisionMixin` cause their number is growing and they're generally the same just with different object name

File: src/edap/edap.py [hunk 1]
```diff
@@ -425,13 +425,130 @@ def create_all_franchises(self, source):
...
+
+    def get_team_component_units(self, machine_name):
+        """
+        Get composing franchise and division from team cn
+        Args:
+            team (dict): ldap team
+
+        Returns:
+        """
+        franchise_name, division_name = machine_name.decode('utf-8').split('-', 1)
```
Comment (matejak) [inline src/edap/edap.py]: The class contains the decoding logic, so an encoding logic would be nice as well (e.g. a static method `team_name(franchise_name, division_name`).

**Comment by matejak** [comment]
So far looks good!

File: src/edap/edap.py (hunk 1 above)
Comment (danil-topchiy) [reply src/edap/edap.py]: Added methods to make team `display_name`, `machine_name`

**Comment by matejak** [comment]
Looks great, merging!

**Pull request merged**